### PR TITLE
Autonomous improvements

### PR DESCRIPTION
**Pull request by lmaxwell24**
mostly involving documenting
other notable differences is stopping after the completion of a trajectory

### CODE REVIEW

File: commands/auto/fivebrstandard.py [hunk 1]
```diff
@@ -75,21 +75,25 @@ def __init__(
...
-            WaitCommand(2),
-            FollowTrajectory(drive, pathD),
+            FollowTrajectory(drive, pathC),  # go to terminal, pickup terminal ball
+            WaitCommand(
+                constants.kAutoTerminalWaitTime
+            ),  # wait for user ball in terminal
+            FollowTrajectory(drive, pathD),  # go back to shooting range
             RetractIntake(intake),
-            FeedForward(indexer),
+            FeedForward(indexer),  # shoot balls 4 and 5
```
Comment (LandonBayer) [inline commands/auto/fivebrstandard.py:98]: do we need a pause in between for accuracy?

File: commands/auto/twoblhangerbounce.py [hunk 2]
```diff
@@ -53,10 +53,11 @@ def __init__(
...
-            FollowTrajectory(drive, pathA),
+            FollowTrajectory(drive, pathA),  # pickup ball 2
             RetractIntake(intake),
+            WaitCommand(constants.kAutoTimeFromStopToShoot),
             FeedForward(indexer),
-            WaitCommand(2),
+            WaitCommand(constants.kAutoTimeFromShootToMove),
             HoldBall(indexer),
-            FollowTrajectory(drive, pathB),
+            FollowTrajectory(drive, pathB),  # hit red ball out of way
```
Comment (LandonBayer) [inline commands/auto/twoblhangerbounce.py:62]: Is there a way we can figure out for sure that the large turning is actually the problem with this path?

**Review by LandonBayer — DISMISSED** [review]

File: commands/auto/twoblhangerbounce.py (hunk 2 above)
Comment (lmaxwell24) [reply commands/auto/twoblhangerbounce.py:62]: open the path json in pathweaver

**Review by caloisio — APPROVED** [review]

**Review by LandonBayer — APPROVED** [review]

File: robotcontainer.py [hunk 3]
```diff
@@ -121,6 +125,9 @@ def __init__(self) -> None:
         self.chooser.addOption("Complex Auto", self.complexAuto)
         self.chooser.addOption("Target Auto", self.driveToTarget)
```
Comment (caloisio) [inline robotcontainer.py:126]: We should remove these (and the associated files) if they are no longer used, but that is not a blocker for this PR

**Review by caloisio — APPROVED** [review]

File: robotcontainer.py (hunk 3 above)
Comment (lmaxwell24) [reply robotcontainer.py:126]: A lot of “remove unused” is needed through our codebase right now..

**Pull request merged**